docs/advanced/components/components.md

```diff
@@ -4,13 +4,13 @@ In the process of web developement you want create manageable and maintainable c
 
 ## Why is important the communication between components
 
-Creating a Vuejs app you need to make the components as isolated as they can be, this would make them in reusable and decoupled/loosely coupled shape.
+Creating a Vue.js app you need to make the components as isolated as they can be, this will make them in reusable and decoupled/loosely coupled shape.
```

Comment:
Mejora un poco la redacción de ésta parte.
- _ Creating a Vue.js app you need to make the components as isolated as they can be,_ 
-  _ this will make them in reusable and decoupled/loosely coupled shape._

Reply:
was rewritten

Reply:
I think if you read it twice it makes sense, but we can for sure give it a chance and make it more clear.


Use connectors such as “When creating a Vue.js app...”

Also, “...you need to build/code them as stateless as possible...”

Finally, “this will allow them to be reusable and have a more decoupled(loosely coupled ) shape.